hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdiff/TestRocksDBCheckpointDiffer.java

```diff
@@ -29,60 +31,114 @@
 import java.util.Random;
 import java.util.stream.Collectors;
 
+import org.apache.ozone.rocksdiff.RocksDBCheckpointDiffer.DifferSnapshotInfo;
+import org.apache.ozone.test.GenericTestUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.rocksdb.Checkpoint;
 import org.rocksdb.ColumnFamilyDescriptor;
 import org.rocksdb.ColumnFamilyHandle;
 import org.rocksdb.ColumnFamilyOptions;
-import org.rocksdb.CompressionType;
 import org.rocksdb.DBOptions;
+import org.rocksdb.FlushOptions;
 import org.rocksdb.LiveFileMetaData;
 import org.rocksdb.Options;
 import org.rocksdb.RocksDB;
 import org.rocksdb.RocksDBException;
 import org.rocksdb.RocksIterator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
 
-////CHECKSTYLE:OFF
+/**
+ * Test RocksDBCheckpointDiffer basic functionality.
+ */
 public class TestRocksDBCheckpointDiffer {
 
-  private static final String dbPath   = "./rocksdb-data";
-  private static final int NUM_ROW = 25000000;
-  private static final int SNAPSHOT_EVERY_SO_MANY_KEYS = 999999;
+  private static final Logger LOG =
+      LoggerFactory.getLogger(TestRocksDBCheckpointDiffer.class);
+
+  /**
+   * RocksDB path for the test.
+   */
+  private static final String TEST_DB_PATH = "./rocksdb-data";
+  private static final int NUM_ROW = 250000;
+  private static final int SNAPSHOT_EVERY_SO_MANY_KEYS = 49999;
+
+  /**
+   * RocksDB checkpoint path prefix.
+   */
+  private static final String CP_PATH_PREFIX = "rocksdb-cp-";
+  private final ArrayList<DifferSnapshotInfo> snapshots = new ArrayList<>();
+
+  /**
+   * Graph type.
+   */
+  enum GType {
+    FNAME,
+    KEYSIZE,
+    CUMUTATIVE_SIZE
+  }
+
+  @BeforeAll
+  public static void init() {
+    // Set differ log level
+    GenericTestUtils.setLogLevel(RocksDBCheckpointDiffer.getLog(), Level.INFO);
+    // Set test class log level
+    GenericTestUtils.setLogLevel(TestRocksDBCheckpointDiffer.LOG, Level.INFO);
+  }
 
-  // keeps track of all the snapshots created so far.
-  private static int lastSnapshotCounter = 0;
-  private static String lastSnapshotPrefix = "snap_id_";
+  @Test
```

Comment:
This test doesn't seem to check the results.  Is that going to happen in a future pr?

Reply:
Thanks. Added assertions.